src/components/NotFoundPage.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+export default class NotFoundPage extends Component {
+  componentDidMount() {}
+
+  render() {
+    return (
+      <div>
```

Comment:
Pretty lame ATM.  I stole the language from the main edx.org 404 page.

Reply:
Pretty sure you also promised cats.

Reply:
Crap!